src/scorecard.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+class Scorecard  {
+
+  constructor() {
+    this.frames = [new Frame()];
+    this.currentFrameIndex = 0;
+  }
+
+  play(score) {
+    if(this.gameOver()) {
+      return;
+    } else {
```

Comment:
We can actually remove the `else` block in this case to make the code look cleaner:
```
if(this.gameOver()) {
      return;
}
this.gameSetup();
this.currentFrame().bowl(score);
this.addBonus(score);
```